src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K,V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_SIZE];
```

Comment:
I'd recommend you to move this initialization to your method grow. In this case you'll be have the ability to create your HashMap with internal table= null. See the video "HashMap resize" (3:35)!